### PR TITLE
chore: test deployment pipeline for webbsite

### PR DESCRIPTION
**Pull request by dutterbutter**
## Summary of changes
- Deploys webbsite to netlify for development purposes 
- Adds github workflow for webbsite
- Adds code of conduct 

To preview:
- https://webbsite-dev.netlify.app/

Tasks Remaining for Webb marketing site:
- https://github.com/webb-tools/webb-dapp/issues/766

### Reference issue to close (if applicable)
- Closes 

-----
### Code Checklist 

- [ ] Tested
- [ ] Documented

### CODE REVIEW

File: apps/webbsite/next.config.js [hunk 1]
```diff
@@ -12,6 +12,9 @@ const nextConfig = {
     // See: https://github.com/gregberge/svgr
     svgr: true,
   },
+  images: {
+    unoptimized: true,
+  },
```
Comment (AtelyPham) [inline apps/webbsite/next.config.js:17]: Why do we want unoptimized images here? @dutterbutter

Comment (dutterbutter) [reply apps/webbsite/next.config.js:17]: @AtelyPham good eye. It is a requirement for deploying static Next.js websites. You can view the unsupported features for static exports here in their docs. https://nextjs.org/docs/advanced-features/static-html-export#unsupported-features

Comment (dutterbutter) [reply apps/webbsite/next.config.js:17]: @AtelyPham we can make use of a custom loader to optimize these images just not the default one as mentioned in the supported features part of the docs: https://nextjs.org/docs/advanced-features/static-html-export#supported-features

**Review by AtelyPham — APPROVED** [review]
LGTM 👍

**Pull request merged**